src/server/getters/getLastActiveHackathonId.ts

```diff
@@ -0,0 +1,19 @@
+import { PrismaClient } from "@prisma/client";
+
+const getLastActiveHackathonId = async (
+  prisma: PrismaClient
+): Promise<number | null> => {
+  const hackathons = await prisma.hackathon.findMany();
+
+  // If no hackathons have been organized yet
+  if (hackathons.length === 0) return null;
+
+  const latestHackathon = hackathons.sort(
```

Comment:
You can perform this sorting directly inside the findMany function from Prisma which results in sort by directly in database. This should be more performant and easier to read. Checkout the syntax [here](https://www.prisma.io/docs/orm/prisma-client/queries/filtering-and-sorting)

Reply:
Okay I will check it, but I am not sure it will be faster, because even though it is database it is SqlLite